### ISSUE TITLE
add.significance.stars seems to work incorrectly

### ISSUE THREAD

**Issue by RomanTsegelskyi** [Rapporter/pander#200]
Not a major issue, but seems that `add.significance.starts` option for `summary.lm` is not correct

```r
> ctl <- c(4.17,5.58,5.18,6.11,4.50,4.61,5.17,4.53,5.33,5.14)
> trt <- c(4.81,4.17,4.41,3.59,5.87,3.83,6.03,4.89,4.32,4.69)
> group <- gl(2, 10, 20, labels = c("Ctl","Trt"))
> weight <- c(ctl, trt)
> lm.D9 <- lm(weight ~ group)
> pander(summary(lm.D9), add.significance.stars = TRUE)

---------------------------------------------------------------------
     &nbsp;        Estimate   Std. Error   t value      Pr(>|t|)     
----------------- ---------- ------------ --------- -----------------
   **group1**      -0.1855    0.1557174   -1.19126       _0.249_     

 **(Intercept)**    4.8465    0.1557174   31.12368  _4.185e-17_ * * *
---------------------------------------------------------------------


-------------------------------------------------------------
 Observations   Residual Std. Error   $R^2$   Adjusted $R^2$ 
-------------- --------------------- ------- ----------------
      20              0.6964         0.07308     0.02158     
-------------------------------------------------------------

Table: Fitting linear model: weight ~ group
```

**Comment by daroczig** [comment]
Right, this extra emphasis is not needed, pls merge @201

**Comment by RomanTsegelskyi** [comment]
Fixed by #201
